src/tutorials/0001-data-structures/05.md

```diff
@@ -54,7 +54,7 @@ these data structures backwards, from the leaf nodes on up to the root node.
 
 ## Directed Acyclic Graphs (DAG)
 
-![Directed Acycil Graphs](tutorial-assets/T0001L01-dag.svg)
+![Directed Acycil Graphs](/tutorial-assets/T0001L01-dag.svg)
```

Comment:
@zebateira Can you please change spelling to Acyclic in the alt tag?

Reply:
done 👍